tools/raw-log-converter.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/env python3
+#!/usr/bin/env python3
```

Comment:
can you send this as separate patch please, this one is just a bug that needs to be fixed anyway.